packages/website/utils/sidebarLinks.json

```
@@ -97,6 +97,11 @@
       "title": "Popover",
       "slug": "/components/popover/"
     },
+    {
+      "title": "ProgressStepper",
+      "slug": "/components/progress-stepper/",
+      "status": "stable"
```

If I remember correctly the JSON is generated automatically but it should be the following

```suggestion
      "status": "alpha"
```

---

Great catch here @cf-remylenoir. We have added an updated build here 👍